client-app/pages/catalog.vue

```diff
@@ -325,11 +325,10 @@ import { useRoute } from "vue-router";
 import {
   buildBreadcrumbs,
   DEFAULT_PAGE_SIZE,
-  FacetItem,
-  FacetValueItem,
+  FacetItemType,
+  FacetValueItemType,
```

Comment:
I thought we gonna use separated imports like in [quotes.vue](https://github.com/VirtoCommerce/vc-theme-b2b-vue/pull/463/files/1f935acc008a76c0b1ff33a4b61227d473627a45#diff-2c685f40a20748d414ae1dad4ecc25a92b60a6de7bde8a06d59b68a4c95c678dR161)

Reply:
This PR refers to the "core" folder.  Next we will make PRs for other folders.